src/main/java/yoda/components/Replies.java

```diff
@@ -0,0 +1,39 @@
+package yoda.components;
+
+public class Replies {
+    private final String chatbotName;
```

Comment:
```suggestion
    private final String CHATBOT_NAME;
```
constants should be uppercase with underscore to separate words.